jscomp/bsb/bsb_config_parse.ml

```diff
@@ -146,6 +145,46 @@ let extract_reason_react_jsx (map : json_map) =
   |> ignore;
   !default
 
+let extract_react_jsx (map : json_map) =
```

Comment:
I think this should be called `extract_jsx_version`.

Reply:
Fixed https://github.com/rescript-lang/rescript-compiler/pull/5484/commits/81029b30f20e8b12b9dff1229228e31daa5e7c5b